src/gui/widgets/EffectRackView.cpp

```diff
@@ -138,6 +138,10 @@ void EffectRackView::deletePlugin( EffectView* view )
 	Effect * e = view->effect();
 	m_effectViews.erase( qFind( m_effectViews.begin(), m_effectViews.end(), view ) );
 	delete view;
+	if( m_effectViews.size() == 0 )
+	{
+		fxChain()->m_enabledModel.setValue( false );
+	}
```

Comment:
Since `EffectChain::clear` already have the same functionality, I think you can implement this in `EffectChain::removeEffect` instead.

Reply:
Done in https://github.com/LMMS/lmms/pull/4297/commits/d3e65cad0bc1aff9df7fa286172ed371f387c295